images/kcp-registrar/syncer-patch

```diff
@@ -0,0 +1,8 @@
+        securityContext:
```

Comment:
Shouldn't this have a .yaml extension?

Reply:
It is not required. 
This is just the input for sed command as you can see [here](https://github.com/openshift-pipelines/pipeline-service/commit/8868e632e8c9180e0adb286d1f93017dbdad1a22#diff-010d75c07c9fda576ad68d1bf93347cf3ded384b058db9926fcd2e52ceafb308R218)
Unfortunately I did not find a better way to patch the syncer output  (deployment), not sure if kustomize(as a cleaner alternative) would work here

Reply:
Maybe yq can help? This bit seems a bit hacky :)

Reply:
Also need to add a copy command in the Dockerfile to copy the patch file.

Reply:
Done
I have also added the yml extension since it is now used by `yq`